src/struct_loader.rs

```diff
@@ -59,6 +60,21 @@ where
     named_records: Option<Dict<T>>,
 }
 
+impl<'a, T> IntoIterator for &'a StructLoader<T>
+where
+    T: DeserializeOwned,
+{
+    type Item = (&'a String, &'a T);
+    type IntoIter = Box<dyn Iterator<Item = Self::Item> + 'a>;
+
+    fn into_iter(self) -> Self::IntoIter {
+        match self.named_records.as_ref() {
```

Comment:
As written in the earlier comment, the direct access to the records is deliberately restricted via `get_all_records()` method currently. (of course I'm happy to know if there is any strong use-cases that this is needed :)